UI/PauseScreen.cpp

```diff
@@ -261,8 +261,14 @@ void GamePauseScreen::update() {
 	SetVRAppMode(VRAppMode::VR_MENU_MODE);
 }
 
+GamePauseScreen::GamePauseScreen(const Path &filename)
+	: UIDialogScreenWithGameBackground(filename) {
+	g_OSD.SetShowSidebar(false);
+}
+
 GamePauseScreen::~GamePauseScreen() {
 	__DisplaySetWasPaused();
+	g_OSD.SetShowSidebar(true);
```

Comment:
Technically, on Windows, you can open the GameSettingsMenu or some other things via the File menu, which does also pause things.

Maybe it'd be simpler and more accurate to just have it be a timer, and have EmuScreen set it every frame.  Then the sidebar could all fade out or stop showing or something.

-[Unknown]

Reply:
Yeah, that'd be more robust indeed. Will do.